manual/guides/dualboot.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: manual
+title: Dual Booting Windows and Ubuntu
+subtitle: "A Helpful List of Resources"
+permalink: /manual/guides/dualboot
+---
+
+
+## Why Dual Boot
+
+Dual booting simply means installing two operating systems on one hard drive.
+
+In your career as a computational scientist, you will
+quickly outgrow the limited capabilities of Windows command prompt.
+Why Linux is better for programming:
+1. Compiling code like C++ or SQL is difficult in Windows.
+2. Linux terminal is much more powerful than Windows command prompt
+3. Native support for SSH
+4. Linux is "light-weight" and therefore faster than Windows.
+
```

Comment:
```suggestion

## Why Not Dual Boot

If your preferred device is UNIX-based, like an apple macbook, there is often no need
to dual boot, since everything that can be done on a Linux machine is usually also
possible on a mac. Such an approach has the benefit of also supporting the 
Microsoft Office Suite (which works on a mac too).

If you don't think you'll ever need the Microsoft Office Suite, or the small number
of other commercial software applications which are a challenge to install on a Linux 
machine, you can certainly abandon Windows entirely and install Linux alone.
```